test/1_stdlib/CollectionOfOne.swift

```diff
@@ -1,36 +1,20 @@
-//===--- CollectionOfOne.swift - Tests ------------------------------------===//
-//
-// This source file is part of the Swift.org open source project
-//
-// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See http://swift.org/LICENSE.txt for license information
-// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-//===----------------------------------------------------------------------===//
-// RUN: %target-run-simple-swift | FileCheck %s
+// RUN: %target-run-simple-swift
 // REQUIRES: executable_test
+// REQUIRES: objc_interop
 
-print("testing...")
-// CHECK: testing...
+import Foundation
+import StdlibUnittest
 
-print("for loop: ", terminator: "")
-for x in CollectionOfOne(2) {
-  print(x, terminator: "")
-}
-print(".")
-// CHECK-NEXT: for loop: 2.
-
-let twentyOne = CollectionOfOne(21)
+let OneTests = TestSuite("CollectionOfOne")
+OneTests.test("Basic tests") {
+  for x in CollectionOfOne(2) {
+    expectEqual(2, x)
+  }
```

Comment:
The test used to check not only the values, but also that the loop only did one iteration -- could you add a counter as well?